### PR TITLE
Rule Updates

### PR DESCRIPTION
**Pull request by sidforhbc**
Simplified some rules and changes indentation.

### CODE REVIEW

File: rules/stylistic-issues.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ module.exports = {
         'func-style': ['error', 'declaration', { 'allowArrowFunctions': true }],
 
         // enforce consistent line breaks inside function parentheses
-        'function-paren-newline': ['error', 'never'],
+        'function-paren-newline': ['error', 'multiline'],
```
Comment (track0x1) [inline rules/stylistic-issues.js]: Have we ever had to enforce this rule? I've never seen multiline function params in any of our codebases.

File: rules/best-practices.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ module.exports = {
         'complexity': 'off',
 
         // require return statements to either always or never specify values
-        'consistent-return': 'error',
+        'consistent-return': 'off',
```
Comment (track0x1) [inline rules/best-practices.js]: I think this is a helpful rule that can prevents silly coding mistakes

File: rules/stylistic-issues.js (hunk 1 above)
Comment (sidforhbc) [reply rules/stylistic-issues.js]: header has a bunch :/

File: rules/react.js [hunk 3]
```diff
@@ -274,10 +271,6 @@ module.exports = {
         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md
         'react/jsx-sort-props': 'off',
 
-        // Enforce spaces before the closing bracket of self-closing JSX elements
-        // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-space-before-closing.md
-        'react/jsx-space-before-closing': ['error', 'always'],
```
[the remark this answers is not in the document]
Comment (sidforhbc) [reply rules/react.js]: deprecated

File: rules/stylistic-issues.js [hunk 4]
```diff
@@ -65,7 +65,7 @@ module.exports = {
         'id-match': 'off',
 
         // this option sets a specific tab width for your code
-        'indent': ['error', 4, { 'SwitchCase': 1, 'VariableDeclarator': 1 }],
+        'indent': ['error', 2, { 'SwitchCase': 1, 'VariableDeclarator': 1 }],
```
[the remark this answers is not in the document]
Comment (sidforhbc) [reply rules/stylistic-issues.js]: I will ask but it counts towards the line length limit :)

**Pull request merged**